config/longcheer-l9360.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+OPTIONS=(-r vsn -r vsp)
+PANELS=(
+	[truly_r63350_1080p_video]="longcheer,truly-r63350"
```

Comment:
```suggestion
	[truly_r63350_1080p_video]="longcheer,l9360-r63350"
```
You can add truly as a postfix if you want